### ISSUE TITLE
[IntelliJ][ReportedByUser] MS Azure Explorer - Error Loading gyystorage(default)

### ISSUE THREAD

**Issue by v-xiaocw** [Microsoft/azure-tools-for-java#2747]
Build: Maket build

Repro Steps:
1.Right-click on HDInsight
2.Celect refresh

Result:
IntelliJ build version: 2018.3.1 IC-183.4588.27
OS: Windows 10
JDK: JetBrains s.r.o 1.8.0_152-release
Plugin version: 3.18.0-2018.3
Error message: 
```
com.microsoft.azure.storage.StorageException: This request is not authorized to perform this operation.
	at com.microsoft.azure.storage.StorageException.translateException(StorageException.java:89)
	at com.microsoft.azure.storage.core.StorageRequest.materializeException(StorageRequest.java:315)
	at com.microsoft.azure.storage.core.ExecutionEngine.executeWithRetry(ExecutionEngine.java:175)
	at com.microsoft.azure.storage.core.LazySegmentedIterator.hasNext(LazySegmentedIterator.java:109)
	at java.util.Iterator.forEachRemaining(Iterator.java:115)
	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
	at com.microsoft.azure.hdinsight.serverexplore.hdinsightnode.StorageAccountNode.refreshItems(StorageAccountNode.java:111)
	at com.microsoft.tooling.msservices.serviceexplorer.RefreshableNode.refreshItems(RefreshableNode.java:94)
	at com.microsoft.tooling.msservices.serviceexplorer.RefreshableNode$2.run(RefreshableNode.java:160)
	at com.microsoft.intellij.helpers.IDEHelperImpl$2$1.run(IDEHelperImpl.java:157)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:727)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:164)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:582)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:87)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:151)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:403)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:314)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by wezhang** [comment]
@v-xiaocw , which node did you operate at?

**Comment by v-xiaocw** [comment]
@wezhang 
I operated at "HDInsight"

**Comment by t-rufang** [comment]
I also encounter this issue when I refresh HDInsight node `gyytestfromcli2`.  The default storage account is an Azure blob storage account named [`gyystorage`](https://ms.portal.azure.com/#@72f988bf-86f1-41af-91ab-2d7cd011db47/resource/subscriptions/7f31cba8-b597-4129-b158-8f21a7395bd0/resourceGroups/gyytest/providers/Microsoft.Storage/storageAccounts/gyystorage/overview). Got the following error message when I want to check the details of the storage account in portal(debug mode).
```xml
<?xml version="1.0" encoding="utf-8"?><Error><Code>AuthorizationFailure</Code><Message>This request is not authorized to perform this operation.
RequestId:7edf045d-a01e-0049-5c12-d0d5d3000000
Time:2019-03-01T09:36:17.6662649Z</Message></Error>
```

Therefore, what I try to tell is that the error is not caused by our tool but the azure storage account.

**Comment by wezhang** [comment]
@t-rufang , let's swallow the exception at the `HDInsight` node refreshing.

**Comment by t-rufang** [comment]
Fixed in #2822

**Comment by v-xiaocw** [comment]
Fixed with build 1016
